ui/app/src/state/epics/misc.ts

```diff
@@ -91,7 +91,8 @@ const epics = [
 				const destinationPath = editContentTypeTemplate.type === type ? getParentPath(path) : payload.path;
 				return merge(
 					of(blockUI({ message: getIntl().formatMessage(translations.verifyingAffectedWorkflows) })),
-					fetchSandboxItem(state.sites.active, path).pipe(
+					fetchContentItems(state.sites.active, [path]).pipe(
+						map((items) => items[0]),
```

Comment:
Use single retrieval API...

**Check throughly that this isn't happening in multiple places.** Check all occurances of `fetchContentItems` to determine if they are truely bulk calls and update to single if not.